CP77.CR2W/WolvenKit.Cyberformats.csproj

```diff
@@ -11,9 +11,9 @@
 		<RepositoryType>git</RepositoryType>
 		<RepositoryUrl>https://github.com/WolvenKit/Wolven-kit</RepositoryUrl>
 		<PackageTags>wolvenkit, cyberpunk2077</PackageTags>
-		<AssemblyVersion>1.0.5.0</AssemblyVersion>
-		<FileVersion>1.0.5.0</FileVersion>
-		<Version>1.0.5</Version>
+		<AssemblyVersion>1.0.6.0</AssemblyVersion>
+		<FileVersion>1.0.6.0</FileVersion>
```

Comment:
We should discuss how to version all the projects soon, semver would be the best IMO, we nearly do that already but not always